src/Compute/Compute/help/New-AzVmssConfig.md

```diff
@@ -773,6 +805,21 @@ Accept pipeline input: True (ByPropertyName)
 Accept wildcard characters: False
 ```
 
+### -ProgressAction
```

Comment:
Looks like this bug is back, the PS team will need to remove this or you should.

Reply:
removed